.sun-ci.yml

```
@@ -0,0 +1,90 @@
+default:
+  workspace: shared
+
+stages:
+  - build
+  - check
+
+jobs:
+  - name: Build Laravel
+    stage: build
+    image: ghcr.io/avengerscodelovers/php:8.2-ci
+    services:
```

```suggestion
    services: &commonServices
```
Thử xem có tận dụng lại được block giống nhau bên trong file yml xem seo
https://gist.github.com/vitaLee/7803420